ntbs-service/Migrations/NtbsContextModelSnapshot.cs

```diff
@@ -15791,6 +15811,8 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .IsUnique()
                         .HasFilter("[ServiceAdGroup] IS NOT NULL");
 
+                    b.HasIndex("IsLegacy", "Name");

```

Comment:
@jasiekmiko @conorsheehansw I saw this on I think Conor's migration a couple of days ago as well - it doesn't look like this is applied on test/int but it's been added to the snapshot but not the migration here, might need a sanity check to make sure it's getting added somewhere

Reply:
Yes, it was my omission, but I think Conor has already picked up on. As long as @conorsheehansw's change is in/almost in, we won't need this one, but it'll probably get picked up in the merge either way :)